strr-web/components/bcros/status-card/StatusCard.vue

```diff
@@ -70,11 +71,30 @@ const {
   isSingle: boolean,
 }>()
 
-const { registrationId, registrationNumber, status, registrationStatus } = applicationHeader
+const { registrationId, registrationNumber, status, hostStatus, examinerStatus, registrationStatus } = applicationHeader
 const applicationId = applicationHeader.id.toString()
-const flavour = getChipFlavour(status)
+const flavour = computed(() => {
```

Comment:
I think there could be a mistake here. It should be kept as is, `const flavour = getChipFlavour(status)`, because hostStatus or examinerStatus are serialized string statuses returned by API (see below) but getChipFlavour() needs an enum status. 
```
application_dict["header"]["hostStatus"] = ApplicationSerializer.HOST_STATUSES.get(application.status, "")
application_dict["header"]["examinerStatus"] = ApplicationSerializer.EXAMINER_STATUSES.get(application.status, "")
```

Reply:
Discussed